docs/get-invited-and-onboard-to-techpass.md

```diff
@@ -127,33 +117,31 @@ You need the following to get invited to TechPass and complete the onboarding:
 
   ![vendor-mfa-9](assets/support/vendor-mfa-9-new.png)
 
-  Now you will be prompted to reset your initial password.
-</details>
+  Now you are now prompted to reset your initial password.
 
-## Step 4. Reset initial password
 
-<details> <summary style="font-size:20px;font-weight:bold"> Reset the initial password</summary>
```

Comment:
i thought it should read "Reset your initial password".

Like "Change your default security info method"
https://support.microsoft.com/en-us/account-billing/set-up-text-messaging-as-your-verification-method-a61dcc0c-a003-424f-8a64-d92534ba0dad

But i may not make sense here

Reply:
done.